### PR TITLE
ebv portal queries

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Die Portal-API von GEO BON anbinden](https://trello.com/c/j9nmED8g/580-die-portal-api-von-geo-bon-anbinden)

File: services/src/handlers/ebv.rs [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+        .await?;
+
+    let dataset: Option<EbvDataset> = response
+        .data
+        .into_iter()
+        .map(|data| EbvDataset {
+            id: data.id,
+            name: data.title,
+            author: format!(
+                "{name} ({institution})",
```
Comment (michaelmattig) [inline services/src/handlers/ebv.rs:191]: why do you flatten the data?

File: services/src/handlers/ebv.rs [hunk 2]
```diff
@@ -0,0 +1,671 @@
...
+    use geoengine_datatypes::util::test::TestDefault;
+    use httptest::{matchers::request, responders::status_code, Expectation};
+    use serde_json::json;
+
+    async fn send_test_request<C: SimpleContext>(
+        req: test::TestRequest,
+        ctx: C,
+        mock_address: String,
+    ) -> ServiceResponse {
+        // let mock_address = mock_address.trim_end_matches(|c| c == '/').to_owned();
```
Comment (michaelmattig) [inline services/src/handlers/ebv.rs:224]: remove comment

File: services/src/handlers/ebv.rs [hunk 3]
```diff
@@ -0,0 +1,671 @@
...
+async fn get_classes<C: Context>(
+    _params: web::Query<()>,
+    base_url: web::Data<BaseUrl>,
+    _session: C::Session,
+    _ctx: web::Data<C>,
+) -> Result<impl Responder> {
+    let base_url = base_url.get_ref().as_ref();
+    let url = format!("{base_url}/ebv");
+
+    dbg!(&url);
```
Comment (michaelmattig) [inline services/src/handlers/ebv.rs:101]: remove

File: services/src/handlers/ebv.rs [hunk 4]
```diff
@@ -0,0 +1,671 @@
...
+
+    let response = reqwest::get(url)
```
Comment (michaelmattig) [inline services/src/handlers/ebv.rs:103]: maybe log the request (and response code?) s.t. we have an easier time debugging. The same for the other requests

File: services/src/handlers/ebv.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply services/src/handlers/ebv.rs]: bd2aade

File: services/src/handlers/ebv.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply services/src/handlers/ebv.rs]: [6e23b61](https://github.com/geo-engine/geoengine/pull/466/commits/6e23b6194688c36fbcfabf1f2d39daf4b9d5021d)

File: services/src/handlers/ebv.rs (hunk 3 above)
Comment (ChristianBeilschmidt) [reply services/src/handlers/ebv.rs]: 6e23b61

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**